### PR TITLE
data:ayush-3150

### PR DESCRIPTION
**Pull request by ayush-3150**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/4210"></a>

### CODE REVIEW

**Review by Sindhuinti — APPROVED** [review]
Looks good 👍🏻

File: data/ayush-3150.json [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    },
+    {
+      "name": "Linkedin",
+      "url": "https://www.linkedin.com/in/ayush-gandhi-8443a8176/",
+      "icon": "FaLinkedin"
+    },
+    {
+      "name": "Youtube",
+      "url": "https://www.youtube.com/channel/UCYpQICoB6LNdHRyvpPSuILw",
+      "icon": "FaYouTube"
```
Comment (Aadarsh805) [inline data/ayush-3150.json:26]: All good, just fix this youtube icon, use `FaYoutube`

**Review by Aadarsh805 — CHANGES_REQUESTED** [review]

File: data/ayush-3150.json [hunk 2]
```diff
@@ -0,0 +1,39 @@
+{
+  "name": "Ayush Gandhi",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "I am a Software Engineer. A noob who always try to learn new things!",
+  "avatar":"https://avatars.githubusercontent.com/u/60433144?s=400&u=1945de7a9bb08a5ed799907763f2367285e77e80&v=4",
```
Comment (Pradumnasaraf) [inline data/ayush-3150.json:6]:
```suggestion
```

**Review by Pradumnasaraf — CHANGES_REQUESTED** [review]
Hey,

In the recent update, we don't need to add `avatar:` anymore, it will auto-pick your GitHub profile picture. So, please remove that.
Updated docs: https://linkfree.eddiehub.io/docs/quickstart

**Comment by ayush-3150** [comment]
First of all, Thank you for the time to review my pull request and for giving me comments. I have removed Avatar and fixed the youtube icon typo.

**Review by Panquesito7 — APPROVED** [review]
Thanks! 🎉

**Pull request merged**